### PR TITLE
Added path validation to string inputs

### PR DESCRIPTION
**Pull request by aramesh7**
Includes some required formatting changes

### CODE REVIEW

File: model_analyzer/config/input/config_command_profile.py [hunk 1]
```diff
@@ -312,33 +303,33 @@ def objective_list_output_mapper(objectives):
         constraints_scheme = ConfigObject(
             schema={
                 'perf_throughput':
-                ConfigObject(schema={
-                    'min': ConfigPrimitive(int),
-                }),
+                    ConfigObject(schema={
```
Comment (Tabrizian) [inline model_analyzer/config/input/config_command_profile.py:306]: Is this PR formatted using format.py?

Comment (Tabrizian) [reply model_analyzer/config/input/config_command_profile.py:306]: Guan, created a script to autoformat using format.py in a git commit. https://github.com/triton-inference-server/common/blob/main/tools/pre-commit

File: model_analyzer/config/input/config_defaults.py [hunk 2]
```diff
@@ -12,11 +12,13 @@
...
 # limitations under the License.
 
+import os
+
 #
 # Common defaults
 #
 
-DEFAULT_CHECKPOINT_DIRECTORY = './checkpoints'
+DEFAULT_CHECKPOINT_DIRECTORY = os.path.join(os.getcwd(), 'checkpoints')
```
Comment (Tabrizian) [inline model_analyzer/config/input/config_defaults.py:21]: Does this mean that relative directories no longer work?

**Review by Tabrizian — COMMENTED** [review]
LGTM, please only format the PR.

**Review by Tabrizian — APPROVED** [review]

File: model_analyzer/config/input/config_defaults.py (hunk 2 above)
Comment (aramesh7) [reply model_analyzer/config/input/config_defaults.py:21]: Actually some of the qa tests test this (L0_export_metrics)

Comment (aramesh7) [reply model_analyzer/config/input/config_defaults.py:21]: I added some code to make sure.

**Review by Tabrizian — DISMISSED** [review]

**Review by Tabrizian — APPROVED** [review]

**Pull request merged**